Src/UnitTests/MailerUnitTests/Mail/CustomMailerTests.cs

```diff
@@ -15,7 +15,7 @@ public class CustomMailerTests
         [Fact]
         public async Task CustomMailerSucessful()
         {
-            async Task SendMailCustom(string message, string subject, IEnumerable<MailRecipient> to, MailRecipient from, MailRecipient replyTo, IEnumerable<MailRecipient> cc, IEnumerable<MailRecipient> bcc, IEnumerable<Attachment> attachments)
+            async Task SendMailCustom(string message, string subject, IEnumerable<MailRecipient> to, MailRecipient from, MailRecipient replyTo, MailRecipient sender, IEnumerable<MailRecipient> cc, IEnumerable<MailRecipient> bcc, IEnumerable<Attachment> attachments)
```

Comment:
Could we also add some tests against the `subject` in `GeneralMailTests` and `SmtpMailerTests`?